lib/pbench/server/database/models/server_settings.py

```diff
@@ -200,92 +200,93 @@ class ServerConfig(Database.Base):
 
     @staticmethod
     def _validate(key: str, value: JSONVALUE) -> JSONVALUE:
         try:
-            config = SERVER_CONFIGURATION_OPTIONS[key]
+            setting = SERVER_SETTINGS_OPTIONS[key]
         except KeyError as e:
             if key:
-                raise ServerConfigBadKey(key) from e
+                raise ServerSettingBadKey(key) from e
             else:
-                raise ServerConfigMissingKey() from e
-        return config["validator"](key, value)
+                raise ServerSettingMissingKey() from e
+        return setting["validator"](key, value)
 
     @staticmethod
-    def create(key: str, value: JSONVALUE) -> "ServerConfig":
-        """A simple factory method to construct a new ServerConfig setting and
+    def create(key: str, value: JSONVALUE) -> "ServerSetting":
+        """A simple factory method to construct a new ServerSetting setting and
         add it to the database.
 
         Args:
-            key : config setting key
-            value : config setting value
+            key : server setting key
+            value : server setting value
 
         Returns:
-            A new ServerConfig object initialized with the parameters and added
+            A new ServerSetting object initialized with the settings and added
             to the database.
         """
 
         v = __class__._validate(key, value)
-        config = ServerConfig(key=key, value=v)
-        config.add()
-        return config
+        setting = ServerSetting(key=key, value=v)
+        setting.add()
+        return setting
 
     @staticmethod
-    def get(key: str, use_default: bool = True) -> "ServerConfig":
-        """Return a ServerConfig object with the specified configuration key
-        setting.
+    def get(key: str, use_default: bool = True) -> "ServerSetting":
+        """Return a ServerSetting object with the specified key setting.
 
-        For example, ServerConfig.get("dataset-lifetime").
+        For example, ServerSetting.get("dataset-lifetime").
 
         If the setting has no definition, a default value will optionally
         be provided.
 
         Args:
-            key : System configuration setting name
+            key : System setting name
             use_default : If the DB value is None, return a default
 
         Raises:
-            ServerConfigSqlError : problem interacting with Database
+            ServerSettingSqlError : problem interacting with Database
 
         Returns:
-            ServerConfig object with the specified key name or None
+            ServerSetting object with the specified key name or None
         """
         try:
-            config = Database.db_session.query(ServerConfig).filter_by(key=key).first()
-            if config is None and use_default:
-                config = ServerConfig(key=key, value=__class__._default(key))
+            setting = (
+                Database.db_session.query(ServerSetting).filter_by(key=key).first()
+            )
```

Comment:
Elsewhere, you've split this line instead of letting Black wrap it (and I think those are more readable).  E.g.,
```
            dbs = Database.db_session
            setting = dbs.query(ServerSetting).filter_by(key=key).first()
```

Reply:
Sure.

Reply:
Fixed.